reV/bespoke/bespoke.py

```diff
@@ -969,27 +975,36 @@ def get_lcoe_kwargs(self):
             updated based on the bespoke optimized system_capacity, includes
             fixed_charge_rate, system_capacity (kW), capital_cost ($),
             fixed_operating_cos ($), variable_operating_cost ($/kWh)
+            Data source priority: outputs, plant_optimizer,
+            original_sam_sys_inputs, meta
         """
 
-        if 'system_capacity' not in self.outputs:
-            msg = ('Could not find system_capacity in the outputs, need to '
-                   'run_plant_optimization() to get the optimized '
-                   'system_capacity before calculating LCOE!')
-            logger.error(msg)
-            raise RuntimeError(msg)
-
-        lcoe_kwargs = {
-            'fixed_charge_rate':
-                self.original_sam_sys_inputs['fixed_charge_rate'],
-            'system_capacity': self.plant_optimizer.capacity,
-            'capital_cost': self.plant_optimizer.capital_cost,
-            'fixed_operating_cost': self.plant_optimizer.fixed_operating_cost,
-            'variable_operating_cost':
-                self.plant_optimizer.variable_operating_cost}
+        kwargs_list = ['fixed_charge_rate', 'system_capacity', 'capital_cost',
+                       'fixed_operating_cost', 'variable_operating_cost']
+        lcoe_kwargs = {}
+
+        for kwarg in kwargs_list:
+            if kwarg in self.outputs:
+                lcoe_kwargs[kwarg] = self.outputs[kwarg]
+            elif getattr(self.plant_optimizer, kwarg, None) is not None:
```

Comment:
Look at this wonderful opportunity for our favorite marine mammal!! 

![image](https://github.com/NREL/reV/assets/28815465/fb913c09-f91f-46e0-b3bd-770dc5c8495e)